03-storage/09-object-deletion.md

```diff
@@ -0,0 +1,45 @@
+### Object Deletion
+
+We cannot guarantee permanent object removal in distributed system.
+
+Instead, Storage Node designed to be self-cleaning. Object Garbage Collector component is responsible for objects deletion physically from Blobstors and Metabase. Object Garbage Collector also serves the Control API endpoint for external data controllers and allows them make decisions on object deletion basing on their specific criterias. It runs periodically, classifies objects and selects them for removal.
```

Comment:
Object Garbage Collector component deletes objects physically from Blobstors and Metabase. Plus, it exposes the Control API endpoint to external data controllers and allows them to make decisions on object deletion regarding applicable specific criteria.